add srtp_add_or_update_stream used by telephony/linphone/mediastreamer2

https://gitlab.linphone.org/BC/public/external/srtp/-/commit/c48e27278e656dc4b883044ec85245fb3d22dc51.patch

Index: srtp.def
--- srtp.def.orig
+++ srtp.def
@@ -7,6 +7,7 @@ srtp_unprotect
 srtp_unprotect_mki
 srtp_create
 srtp_add_stream
+srtp_add_or_update_stream
 srtp_remove_stream
 srtp_update
 srtp_update_stream
